templates/shared/_openstack-pie-chart.html

```diff
@@ -1,7 +1,7 @@
 <figure id="ubuntu-pie" class="p-pie"></figure>
 <p class="u-no-margin--bottom"><small>Ubuntu powers 55% of OpenStack in production. Source: <a href="https://www.openstack.org/assets/survey/April2017SurveyReport.pdf" class="external">OpenStack survey &mdash; April 2017 [PDF 10.6MB]</a></small></p>
 
-<script src="{{ ASSET_SERVER_URL }}3c99518b-d3-version3.5.6.min.js"></script>
+<script src="{{ TEXT_ASSET_SERVER_URL }}3c99518b-d3-version3.5.6.min.js"></script>
```

Comment:
I doubt this works. I wouldn't bother using a variable for this - just hardcode `https://assets.ubuntu.com/v1/` for these cases.